ecl/hql/hqltrans.cpp

```diff
@@ -375,19 +414,20 @@ static void insertScopeSymbols(IHqlScope * newScope, HqlExprArray const & symbol
 
 //---------------------------------------------------------------------------
 
-static HqlTransformerBase * activeTransformer;      // not thread safe
+static __thread HqlTransformerBase * activeTransformer = nullptr;
```

Comment:
I believe __thread is non-standard.  Specifically, VC++ doesn't suport __thread.  thread_local has been proposed for the the Cx00 standard.  It is supported by both gcc and VC++(https://msdn.microsoft.com/en-us/library/6yh4a9k1.aspx)